geoh5py/objects/octree.py

```diff
@@ -194,18 +194,28 @@ def octree_cells(self) -> Optional[np.ndarray]:
 
     @octree_cells.setter
     def octree_cells(self, value):
+
         if value is not None:
-            value = np.vstack(value)
+            if isinstance(value, np.recarray):
```

Comment:
**question (out-of-scope):** these changes do not seem related to the issue about Root group. Revert from this Pull Request?

Reply:
oh please, I have been so good recently. Please let that one slide, please?

Reply:
alright